micronota/db/uniref.py

```diff
@@ -89,11 +89,14 @@
 from ..util import _overwrite, _download
 
 
+_status = ['Swiss-Prot', 'TrEMBL']
+_kingdom = ['Bacteria', 'Archaea', 'Viruses', 'Eukaryota', 'other']
```

Comment:
anything that do not fit into the first 4.

On Fri, Feb 19, 2016 at 2:30 PM, Daniel McDonald <notifications@github.com>
wrote:

> In micronota/db/uniref.py
> <https://github.com/biocore/micronota/pull/44#discussion_r53528180>:
>
> > @@ -89,11 +89,14 @@
> >  from ..util import _overwrite, _download
> >
> >
> > +_status = ['Swiss-Prot', 'TrEMBL']
> > +_kingdom = ['Bacteria', 'Archaea', 'Viruses', 'Eukaryota', 'other']
>
> what's other?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/44/files#r53528180>.
>